src/server.c

```diff
@@ -3287,6 +3330,24 @@ void blockingOperationEnds() {
     }
 }
 
+int isScriptTimedout() {
+    return server.script_timedout;
+}
```

Comment:
Should this be moved into script.(c|h)? Perhaps a better time for a little more modularity. We can pull the stuff out of server.h as well.

Reply:
Good point, will move it.

Reply:
Actually, re-looking at this, it is looking at the server struct so maybe we should keep it here?

Reply:
Maybe it shouldn't be in server struct at all then?

Reply:
I agree, added another PR and moved the entire timeout handling into `script.c`